browser/iframe-browser.js

```diff
@@ -0,0 +1,106 @@
+define(function (require, exports, module) {
+    "use strict";
+
+    var CommandManager      = brackets.getModule("command/CommandManager"),
+        Menus               = brackets.getModule("command/Menus"),
+        MainViewManager     = brackets.getModule("view/MainViewManager"),
+        Commands            = brackets.getModule("command/Commands"),
+        _panel		        = null;
+
+
+
+    /*
+     * Function that manages initial arguments
+     * html: We are expecting the user to send us a packaged blob object for use
+     *      if undefined we will not fill the iframe with an src
+     * vertical: We are expecting a boolean value, true for vert, false for horizontal
+     *      if vertical, we will display the sidepanel vertically with the html sent
+     *      if horizontal, we will display the sidepanel horizontally with the html sent
+     *      if undefinied, we will default to vertical layout
+     */
+    function browse(html, vertical) {
+        /*
+         *Retrieve Current layout State
+         *result has 2 values in it, row and column
+         *these together will help us get what kind of layout the application is currenlty in
+         */
+
+         /*
+          * Retrieving Current Layout State
+          *     Will return 2 values, row and column
+          *     These refer to the current layout state, (no split, split vert, split hori)
+          */
+        var result = MainViewManager.getLayoutScheme();
+        //If undefined do some checking to figure out what to do
+        if(vertical === undefined) {
```

Comment:
Ye, I guess the way I was checking may have been "efficient" in my eyes, but needlessly complicated